generator/internal/sidekick/sidekick_test.go

```diff
@@ -44,7 +44,7 @@ var (
 func TestRustFromOpenAPI(t *testing.T) {
 	var outDir = fmt.Sprintf("%s/rust/openapi/golden", testdataDir)
 	cmdLine := &CommandLine{
-		Command:             "generate",
+		Command:             []string{},
```

Comment:
Because with this refactor, the CommandLine construction now happens after the command path is interpreted, and its value is ignored during the command execution -- but available, if a command needs it. 
The easiest path to test behavior is what we are doing here, passing a CommandLine struct directly to the runCommand function. Alternatively, we could pass an array of strings to `sidekick.Run` function, but that is more verbose.

Reply:
Ack.  I think my question was less interesting: I am seeing other tests in this file use `Command: []string{"generate"}` and wonder why this test does not do the same.  Should they all be simplified like this test, or should they all use the longer incantation?

If there is a reason why this test should be different, then that deserves a comment.

Reply:
Oh, I see... I missed the other tests in this file. The cmd name in the args is not useful in any of them, so it's just ignored. I have removed this value from the other tests too, so they're all following the same pattern.